tests/api/metrics/test_metrics_spans.py

```diff
@@ -265,4 +298,99 @@ def test_metrics_spans_empty_name_raises_error():
             pass
 
     # then
-    assert err.match("step name must not be empty")
+    assert err.match("span name must not be empty")
+
+
+@mock.patch(
+    "time.monotonic",
+    side_effect=[
+        110535.515,
+        110535.656,
+        110536.093,
+        110536.656,
+        110537.375,
+        110537.89,
+        110538.093,
+        110538.484,
+        110539.046,
+        110539.968,
+        110540.578,
+        110541.218,
+        110541.546,
+        110541.781,
+        110542.687,
+        110542.781,
+        110543.39,
+        110544.359,
+        110545.296,
+        110545.812,
+        110546.265,
+        110546.5,
+        110547.078,
+        110547.984,
+        110548.078,
```

Comment:
I recognize this is a bit of a red flag but I'm unsure how to reconcile this, since otherwise the start_times end up being all 0 when running the tests on windows

Reply:
how about making `side_effect` a function that returns an increasing number? (maybe `side_effect=lambda: *_, **__: next(GLOBAL_COUNT)` would be enough?

Reply:
please, let's not do this, it's not maintainable and generally not best practice. You can definitely just make the clock "tick" as Frank suggested, that should be sufficient.

Reply:
Good call, added a local fixture for it here: https://github.com/snowflakedb/snowflake-cli/pull/1835/files#diff-839165dddafe1dd908ef843269b12e1cd4a7158ea95e5e7a2ee210ea7ec56de0R26-R31